samples/snippets/conftest.py

```diff
@@ -62,14 +42,25 @@ def project_id(bigquery_client):
 
 @pytest.fixture(scope="session")
 def dataset_id(bigquery_client: bigquery.Client, project_id: str):
-    dataset_id = resource_prefix()
+    dataset_id = prefixer.create_prefix()
     full_dataset_id = f"{project_id}.{dataset_id}"
     dataset = bigquery.Dataset(full_dataset_id)
     bigquery_client.create_dataset(dataset)
     yield dataset_id
     bigquery_client.delete_dataset(dataset, delete_contents=True, not_found_ok=True)
 
 
+@pytest.fixture
+def random_table_id(bigquery_client: bigquery.Client, project_id: str, dataset_id: str):
+    """Create a new table ID each time, so random_table_id can be used as
+    target for load jobs.
+    """
+    random_table_id = prefixer.create_prefix() + "_random_table"
```

Comment:
Unclear what value the `_random_table` suffix is giving you other than extra characters.

Reply:
Yeah, not actually necessary. Removed.